.github/workflows/container-build-and-push.yaml

```
@@ -0,0 +1,24 @@
+name: Build and push image
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - id: build-push-image
+        uses: castoredc/container-build-push-action@v1
+        with:
+          image: "cdms/senchacmd"
+          tags: "6.6.0.13"
```

Perhaps good to keep the tags in line with what used to be done via the Makefile?
```suggestion
          tags: "6.6.0.13 6.6.0 6.6"
```

---

Don't think anyone used that Makefile to be honest but good idea :)